common/utils/helpers.py

```
@@ -0,0 +1,38 @@
+    """Base test class with common setup and helper methods."""
+
+    def setUp(self):
+        """Common test setup."""
+        self.factory = APIRequestFactory()
+        self.user1 = User.objects.create_user(username="user1", id=1)
+        self.user2 = User.objects.create_user(username="user2", id=2)
+        self.test_category = InsultCategory.objects.create(
+            category_key="TEST", name="Base Test Class Category"
+        )
+
+    def create_insult(self, **kwargs):
```

Ruff (0.12.2)

Missing return type annotation for public function `create_insult`

(ANN201)